simulation/traffic_simulator/CMakeLists.txt

```diff
@@ -13,6 +13,15 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra -Wpedantic -Werror)
 endif()
 
+# Set ROS_DISTRO macros
```

Comment:
This section was removed because it is no longer needed.
Please see [this thread](https://github.com/tier4/scenario_simulator_v2/pull/924#discussion_r1105697285) for the reason.